### DISCUSSION TITLE
It panic when multiple elements in list

### DISCUSSION THREAD

**Discussion by deepseawhale** [RealAlexandreAI/json-repair#12] [Q&A] [answered]
Panic case:
"```json {  'reviews': [    {      'version': 'new',      'line': 1,      'severity': 'Minor',      'issue_type': 'Standard practice suggestion',      'issue': 'The merge request description is missing a link to the original issue or bug report.',      'suggestions': 'Add a link to the original issue or bug report in the *Issue* section.'    },    {      'version': 'new',      'line': 2,      'severity': 'Minor',      'issue_type': 'Standard practice suggestion',      'issue': 'The merge request description is missing a description of the critical issue or bug being addressed.',      'suggestions': 'Add a description of the critical issue or bug being addressed in the *Problem* section.'    } ]```"
🤔

**Comment by RealAlexandreAI** [comment]
Hello, I am currently focused on a gen ui project, so this case may be delayed by a few days. I will make the fixes during an appropriate holiday, and PRs are also welcome.

**Comment by RealAlexandreAI** [comment]
https://stackoverflow.com/questions/2275359/jquery-parsejson-throws-invalid-json-error-due-to-escaped-single-quote-in-json/2275428#2275428


In the JSON standard, keys for objects must be enclosed in double quotes ("). This is a clear requirement of the JSON format to ensure consistency and correct parsing of data.

For example, a standard JSON object should look like this:

{
  "key": "value"
}
If single quotes (') are used to enclose keys, it will not conform to the standard JSON format and may cause JSON parsers to fail to parse the data correctly. For example, the following format is not acceptable:

{
  'key': 'value'  // Incorrect: the key uses single quotes
}
So, the correct practice is always to use double quotes to define keys in JSON objects.

**Comment by RealAlexandreAI** [comment, marked as answer]
Fixed in v0.0.8.
